kerneldriver_linux.go

```diff
@@ -0,0 +1,68 @@
+// +build linux
```

Comment:
I don't think this distinction is necessary. libusb provides the same interface for all platforms, and the libusb implementation will already signal when a function is not supported. So we can just rely entirely on errors returned by libusb.